react-backend/client/src/components/visitors/thank_you_feedback.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default () => (
```

Comment:
Lol, these thanks components are still almost the same, but I'll let it go at this stage.

Reply:
I was contemplating passing a key to `props` for the component to match on to add the correct copy, but I felt like this wasn't simplifying the already simplified functional component much and would take away from readability, so settled for this 🤷🏽‍♀️. I mean, if it was elm I would have union typed the crap out of it 😏. Happy to implement the above, or any other suggestion if ya want.

Reply:
Nah leave it, just fix the docs.